src/main/src/main/java/org/geoserver/config/util/SecureXStream.java

```diff
@@ -158,6 +203,275 @@ protected MapperWrapper wrapMapper(MapperWrapper next) {
         return new DetailedSecurityExceptionWrapper(next);
     }
 
+    @Override
+    protected void setupConverters() {
+        Mapper mapper = getMapper();
+        ReflectionProvider reflectionProvider = getReflectionProvider();
+        ClassLoaderReference classLoaderReference = getClassLoaderReference();
+        ConverterLookup converterLookup = getConverterLookup();
+
+        registerConverter(new ReflectionConverter(mapper, reflectionProvider), PRIORITY_VERY_LOW);
+        registerConverter(new InternalBlackList(), PRIORITY_LOW);
```

Comment:
If this is a blacklist, should it really be low priority? Shouldn't it be one of the first converters to check.

Reply:
Again, vertatim copy from XStream (the whole method is a copy of the XStream default one, with a minimum amount of adaptations)

Reply:
Okay, I withdraw my concerns then (still seems wrong, but I'm content enough for it to be an XStream problem and not a GeoServer one)